src/components/admin/TableModal/index.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  Button,
+  FormControl,
+  FormLabel,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  NumberInput,
+  NumberInputField,
+  UseDisclosureReturn,
+} from '@chakra-ui/react'
+import React, { useState } from 'react'
+
+type Props = {
+  title: string
+  defaultTableNumber?: string
```

Comment:
Achei que soa melhor, o que acha?
```suggestion
  initialTableNumber?: string
```